calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java

```diff
@@ -970,7 +972,8 @@ public void deleteEventTask(String name, selectViewOption view, selectDayOption
     $(ELEMENT_CONTEXT_MENU_DELETE).click();
     if (isVerify) {
       if (isEvent)
-        alert.verifyAlertMessage(ELEMENT_CONFIRM_DELETE_EVENT_MSG);
+      {alert.verifyAlertMessage(ELEMENT_CONFIRM_DELETE_EVENT_MSG);
+      $(ELEMENT_CONFIRM_DELETE_EVENT_MSG).waitUntil(Condition.disappears,Configuration.timeout);}
```

Comment:
Please format the code with eXo Formatter Settings http://developer.exoplatform.org/#id-formatter-settings